jsignature/forms.py

```diff
@@ -17,6 +17,18 @@ class JSignatureField(Field):
     """
     widget = JSignatureWidget()
 
+    def __init__(self, **options):
+        if 'max_length' in options:
+            # Newer Django versions add 'max_length' by default to a TextField:
+            # https://github.com/django/django/blob/1.10.2/
+            # django/db/models/fields/__init__.py#L2138
+            #
+            # The value is None, we don't use it, and it breaks form field
+            # creation. So remove it.
+            del options['max_length']
```

Comment:
As that code pointer indicates, Django now adds a max_length attribute by default to all TextFields. Since this class inherits TextField, it gets a max_length attribute. But, unless I pop max_length here, the superclass constructor fails because it doesn't expect that arg.

To be honest, while I get what the code does, I can't tell _why_. And I'm not actually certain this is correct or safe, and would love another pair of eyes here.

I can't tell which of https://docs.djangoproject.com/en/1.10/internals/deprecation/ is the relevant deprecation warning myself.